### PR TITLE
Fi 612 fix tests for multi patient

### PR DESCRIPTION
**Pull request by czh-orz**
This PR just adds unit tests back in. It only uses one patient for the tests. 

**Submitter:**
- [X] This pull request describes why these changes were made
- [X] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-612
- [X] Internal ticket links to this PR
- [X] Internal ticket is properly labeled (Community/Program)
- [X] Internal ticket has a justification for its Community/Program label
- [X] Code diff has been reviewed for extraneous/missing code
- [X] Tests are included and test edge cases
- [X] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: generator/uscore/templates/unit_tests/unit_test.rb.erb [hunk 1]
```diff
@@ -12,8 +12,8 @@ describe Inferno::Sequence::<%= class_name %> do
     @token = 'ABC'
     @instance = Inferno::Models::TestingInstance.create(url: @base_url, token: @token, selected_module: '<%= module_name %>')
     @client = FHIR::Client.for_testing_instance(@instance)
-    @patient_id = 'example'
-    @instance.patient_id = @patient_id
+    @patient_ids = 'example'
```
Comment (Jammjammjamm) [inline generator/uscore/templates/unit_tests/unit_test.rb.erb]: Is it possible to replace the other locations where `'example'` is hard coded with this instance variable?

Comment (czh-orz) [reply generator/uscore/templates/unit_tests/unit_test.rb.erb]: Yep good point. All other hardcoded 'example' locations are changed to '@sequence.patient_ids.first'.
